CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# The Skate team is the default owner for the entire codebase.
+*       @mbta/skate-developers
+
+# Lock-files managed by Dependabot do not automatically add reviewers
+mix.lock
+assets/package-lock.json
```

Comment:
**suggestion:** it's not as common, but dependabot does sometimes change `mix.exs` or `package.json` - see #2154 for one recent example - so we should probably include those as well.

Of course this approach in general does have the downside that if a human opens a PR that only changes / adds / removes a dependency they will have to manually request review, but I think that's acceptable.

Reply:
[`5c8c8eb` (#2213)](https://github.com/mbta/skate/pull/2213/commits/5c8c8eb05d808761dbaaf0d281e312c440acb9cb)